Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -43,21 +43,28 @@ public class CropViewController: UIViewController {
     deinit {
         print("CropViewController deinit.")
     }
-    
-    init(config: Mantis.Config = Mantis.Config()) {
+
+    public init(
+        image: UIImage,
+        config: Mantis.Config = Mantis.Config(),
+        cropToolbar: CropToolbarProtocol = CropToolbar(frame: .zero)
+    ) {
         self.config = config
 
         switch config.cropViewConfig.cropShapeType {
         case .circle, .square, .heart:
             self.config.presetFixedRatioType = .alwaysUsingOnePresetFixedRatio(ratio: 1)
         default:
             ()
-        }        
-        
+        }
+
+        cropView = buildCropView(with: image, and: config.cropViewConfig)
```

Comment:
Before I also created CropView inside the CropViewController, recently I refactored code to inject CropView into CropViewController. So CropViewController is not responsible for building CropView now and it is also easier for unit tests. You can check CropViewControllerTests which is using a FakeCropView.

Reply:
If you want to get image, you can use cropView.image inside CropViewController